core/shared/src/main/scala/scalaz/zio/FiberRef.scala

```diff
@@ -34,13 +34,13 @@ import scalaz.zio
  * @param initial
  * @tparam A
  */
-final class FiberRef[A](private[zio] val initial: A) extends Serializable {
+final class FiberRef[A](private[zio] val initial: A) extends AnyVal with Serializable {
```

Comment:
Since we are extending Serializable, it won't help to have `AnyVal`. Also, we are using it as a key in a map, and we want the Object#equals / Object#hashCode. So I'd not extend `AnyVal` here.

Reply:
fixed